source/includes/api/tables/progress-response.rst

```diff
@@ -0,0 +1,62 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 14 66
+
+   * - Field
+     - Type
+     - Description
+
+   * - ``state``
+     - string
+     - The current state of {+c2c-product-name+}.
```

Comment:
Note for review - I'm going to create a ticket to add a page dedicated to the states of the sync tool. Once that page is created, we'll link to it from this description.

Reply:
thanks! that would be a great addition to get the definition of all the states